spec/services/healthcheck_component/sengrid_spec.rb

```diff
@@ -2,34 +2,27 @@
 
 RSpec.describe HealthcheckComponent::Sendgrid, type: :service do
 
-  it_behaves_like 'a healthcheck component'
-
   subject { described_class.new }
 
-  context 'sengrid is available' do
-    let(:successful_smtp_response) do
-      instance_double Net::SMTP::Response, success?: true
-    end
+  let(:response_success) { true }
```

Comment:
Ok, I'll move setting `response_status` inside the context for clarity. Means I'll move `it_behaves_like 'a healthcheck component'` into each context.